lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -135,8 +135,10 @@ describe('ReportRenderer', () => {
       const output = renderer.renderReport(sampleResults, container);
 
       const warningEls = output.querySelectorAll('.lh-run-warnings > ul > li');
-      assert.strictEqual(warningEls.length, 1);
-      assert.ok(/Links.*unsafe/.test(warningEls[0].textContent), 'did not add warning text');
+      assert.strictEqual(warningEls.length, sampleResults.runWarnings.length);
+      for (const warningEl of warningEls) {
+        assert.ok(/Links.*unsafe/.test(warningEl.textContent), 'did not add warning text');
```

Comment:
shouldn't this just be deleted now that warningEls should be length 0?

Reply:
good call.